vendor/bat-native-ledger/src/bat_get_media.cc

```diff
@@ -621,6 +670,24 @@ void BatGetMedia::onFetchPublisherFromDBResponse(ledger::Result result,
                                         _2,
                                         _3));
   } else {
+    if (providerType == TWITCH_MEDIA_TYPE) {
+      std::string twitchHandle = getUserFacingHandle(publisher_blob);
+      std::string twitchFaviconUrl =
+        getFaviconUrl(publisher_blob, twitchHandle);
+      if ((info->name != twitchHandle && !twitchHandle.empty()) ||
+          (info->favicon_url.empty() ||
+          info->favicon_url.find(".invalid") == std::string::npos)) {
+        ledger::VisitData new_visit_data(visit_data);
+        new_visit_data.name = twitchHandle;
```

Comment:
maybe this is a problem for my report, so maybe populate `new_visit_data.name` only if it's not empty and not in general if at the top?

Reply:
Fixed